src/sentry/runner/initializer.py

```diff
@@ -47,6 +49,18 @@ def init_plugin(plugin):
         for cls in plugin.get_custom_contexts() or ():
             contexttype(cls)
 
+    if (hasattr(plugin, 'get_celerybeat_schedule') and plugin.is_enabled()):
+        settings.CELERYBEAT_SCHEDULE.update(plugin.get_celerybeat_schedule())
+
+    if (hasattr(plugin, 'get_celery_imports') and plugin.is_enabled()):
+        settings.CELERY_IMPORTS.append(plugin.get_celery_imports())
+
+    if (hasattr(plugin, 'get_celery_queues') and plugin.is_enabled()):
+        from kombu import Queue
```

Comment:
On second thought, I prefer how it's being done. This way it's a bit more decoupling from kombu.

We could even be smarter here and make `routing_key` optional coming from the plugin since we tend to keep them the same.

We also explicitly set `queue.durable = False`, so we should do there here too.

To be explicit, something like this:

```python
for queue in plugin.get_celery_queues():
    try:
        name, routing_key = queue
    except ValueError:
        name = routing_key = queue
    q = Queue(name, routing_key=routing_key)
    q.durable = False
    settings.CELERY_QUEUES.append(q)
```

This will allow just simply returning back a list of queue names, instead of `(name, routing_key)` tuples for simplification.